docs/commands/New-PSSwaggerModule.md

```diff
@@ -11,15 +11,29 @@ PowerShell command to generate the PowerShell commands for a given RESTful Web S
 
 ## SYNTAX
 
-### SwaggerPath
+### SpecificationPath (Default)
 ```
 New-PSSwaggerModule -SpecificationPath <String> -Path <String> -Name <String> [-Version <Version>]
  [-DefaultCommandPrefix <String>] [-Header <String[]>] [-UseAzureCsharpGenerator] [-NoAssembly]
  [-PowerShellCorePath <String>] [-IncludeCoreFxAssembly] [-InstallToolsForAllUsers] [-TestBuild]
  [-SymbolPath <String>] [-ConfirmBootstrap]
 ```
 
-### SwaggerURI
+### SdkAssemblyWithSpecificationPath
```

Comment:
>SdkAssemblyWithSpecificationPath [](start = 4, length = 32)

What's the plan to get Models namespace?

Reply:
ee36770